CMakeLists.txt

```diff
@@ -11,7 +11,8 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
 set(DATA_DIR \"${CMAKE_INSTALL_PREFIX}/share/osm2pgsql\")
 
 option(BUILD_TESTS "Build test suite" OFF)
-option(WITH_LUA    "Build with lua support" ON)
+option(WITH_LUA    "Build with Lua support" ON)
+option(WITH_LUAJIT "Build with LuaJIT support" OFF)
```

Comment:
How should these be used? `WITH_LUAJIT` implies `WITH_LUA` or not? Or should they be mutually exclusive?

Reply:
Yes, see line 88 + 89 below, `WITH_LUAJIT` implies `WITH_LUA`

Reply:
Well, my reading below is that you need to have both set WITH_LUA and WITH_LUAJIT. That's okay for me though.